tutor/src/components/task-teacher-review/index.cjsx

```diff
@@ -33,21 +34,16 @@ TaskTeacherReview = React.createClass
     crumbKey = if sectionIndex then parseInt(sectionIndex) - 1 else defaultKey
     @setState(currentStep: crumbKey)
```

Comment:
I know you didn't write this, but `parseInt`  must always have it's radix specified, otherwise older browsers will use octal for numbers that lead with a `0` http://stackoverflow.com/questions/6611824/why-do-we-need-to-use-radix